src/index.css

```
@@ -702,22 +702,22 @@
 
   /* Semantic status surfaces */
   .status-info {
-    @apply border border-accent-500/20 bg-accent-500/10 text-[rgb(var(--accent-foreground))] backdrop-blur-xl;
+    @apply border-accent-500/20 bg-accent-500/10 text-[rgb(var(--accent-foreground))] backdrop-blur-xl;
     box-shadow: var(--glass-rim-medium);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)